docs/csharp/programming-guide/generics/benefits-of-generics.md

```diff
@@ -15,7 +15,7 @@ ms.author: "wiwagn"
 # Benefits of Generics (C# Programming Guide)
 Generics provide the solution to a limitation in earlier versions of the common language runtime and the C# language in which generalization is accomplished by casting types to and from the universal base type <xref:System.Object>. By creating a generic class, you can create a collection that is type-safe at compile-time.  
   
- The limitations of using non-generic collection classes can be demonstrated by writing a short program that uses the <xref:System.Collections.ArrayList> collection class from the .NET Framework class library. <xref:System.Collections.ArrayList> is a highly convenient collection class that can be used without modification to store any reference or value type.  
+ The limitations of using non-generic collection classes can be demonstrated by writing a short program that uses the <xref:System.Collections.ArrayList> collection class from the .NET class library. <xref:System.Collections.ArrayList> is a highly convenient collection class that can be used without modification to store any reference or value type.  
```

Comment:
Although it's not part of your PR's intent, @pkulikov, would you mind deleting the phrase "is a highly convenient collection class that"? It sounds suspiciously like a recommendation.

Reply:
@rpetrusha that's true, I've removed that one and then realized that "can be used without modification" also sounds a little bit like a recommendation. So, I've removed that one as well. Please review the change.